dc-build.sh

```diff
@@ -12,4 +12,9 @@ fi
 
 # Building images for all configurations
 # The docker build doesn't supply any environment variables to the Dockerfile, so we can use any profile.
-docker-compose --profile mysql-rabbitmq --profile postgres-redis --env-file ./docker/environments/postgres-redis.env build $1
+
+# From July 2023 Compose V1 stopped receiving updates. Reference: https://docs.docker.com/compose/reference/
+# docker-compose --profile mysql-rabbitmq --profile postgres-redis --env-file ./docker/environments/postgres-redis.env build $1
```

Comment:
Minor nit, but couldn't this line be removed?

Reply:
Hello @dwalleck, did you mean the comments line 16 & 17 both?

Reply:
Yes, that was my thought. I could be talked into there being value in leaving the v1 compose commands for historical purposes.

Reply:
Thanks for the observation. It's remove now.